Dockerfile

```
@@ -0,0 +1,33 @@
+# Используем официальный образ Python
+FROM python:3.12-slim
+
+# Устанавливаем системные зависимости
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    libpq-dev \
+    curl \
+    && rm -rf /var/lib/apt/lists/*
+
+# Устанавливаем poetry
+RUN curl -sSL https://install.python-poetry.org | python3 -
+
+# Обновим переменные среды (чтобы poetry был в PATH)
+ENV PATH="/root/.local/bin:$PATH"
+
+# Устанавливаем рабочую директорию в контейнере
+WORKDIR /app
+
+# Копируем зависимости и README
+COPY pyproject.toml poetry.lock* README.md /app/
+
+# Устанавливаем зависимости без виртуального окружения
+RUN poetry config virtualenvs.create false && poetry install --no-root --no-interaction --no-ansi
```

Устанавливай только основные зависимости - без линтеров
```suggestion
RUN poetry config virtualenvs.create false && poetry install --no-root --no-interaction --no-ansi --only main
```